To: vim_dev@googlegroups.com
Subject: Patch 7.3.350
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.350
Problem:    Block of code after ":lua << EOF" may not work. (Paul Isambert)
Solution:   Recognize the ":lua" command, skip to EOF.
Files:	    src/eval.c


*** ../vim-7.3.349/src/eval.c	2011-09-30 18:35:49.000000000 +0200
--- src/eval.c	2011-10-26 13:12:06.000000000 +0200
***************
*** 20899,20904 ****
--- 20899,20906 ----
  				    && (!ASCII_ISALPHA(p[2]) || p[2] == 'r'))
  			|| (p[0] == 't' && p[1] == 'c'
  				    && (!ASCII_ISALPHA(p[2]) || p[2] == 'l'))
+ 			|| (p[0] == 'l' && p[1] == 'u' && p[2] == 'a'
+ 				    && !ASCII_ISALPHA(p[3]))
  			|| (p[0] == 'r' && p[1] == 'u' && p[2] == 'b'
  				    && (!ASCII_ISALPHA(p[3]) || p[3] == 'y'))
  			|| (p[0] == 'm' && p[1] == 'z'
*** ../vim-7.3.349/src/version.c	2011-10-26 11:44:15.000000000 +0200
--- src/version.c	2011-10-26 13:13:21.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     350,
  /**/

-- 
From "know your smileys":
 :-| :-|   Deja' vu!

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
